lib/src/tasks/format/api.dart

```diff
@@ -23,12 +23,18 @@ import 'package:dart_dev/util.dart' show TaskProcess;
 import 'package:dart_dev/src/tasks/format/config.dart';
 import 'package:dart_dev/src/tasks/task.dart';
 
+// ignore: deprecated_member_use
```

Comment:
It does, otherwise you get a warning about referencing the deprecated argument within its own doc comment...